bundles/org.openhab.binding.knx/README.md

```diff
@@ -197,6 +197,7 @@ If from the KNX bus a `GroupValueRead` telegram is sent to a *-control Channel,
 where parts in brackets `[]` denote optional information.
 
 The optional `<` sign tells whether the group address of the datapoint accepts read requests on the KNX bus (it does, if the sign is there).
+All group addresses marking with `<` and are read by openHAB during startup.
```

Comment:
Looks like "and" is not expected in " All group addresses marking with `<` and are read by"

Reply:
done